scapy/layers/usb.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please don't add that (even know it makes sense), because the file is ASCII anyways (plus it conflicts with https://github.com/secdev/scapy/pull/3655)